### PR TITLE
Scrub Mixpanel data

### PR DESCRIPTION
**Pull request by kcole16**
Removes keys from Mixpanel tracking

### CODE REVIEW

File: src/middleware/index.js [hunk 1]
```diff
@@ -44,12 +44,13 @@ const ACTIONS_TO_TRACK = ['CREATE_NEW_ACCOUNT','ADD_ACCESS_KEY',
...
 
 if (process.env.MIXPANEL_TOKEN) {
-    mixpanel.init(process.env.MIXPANEL_TOKEN)
+    mixpanel.init(process.env.MIXPANEL_TOKEN, {'property_blacklist': ['$current_url']})
 }
 
 const analyticsMiddleware = store => next => action => {
     let details = {
-        pathname: window.location.pathname,
+        pathname: !window.location.pathname.includes('recover-with-link') ?
```
Comment (marcinbodnar) [inline src/middleware/index.js:52]: We can store `recover-with-link` as const, we try to do that with all path related functionalities, like here:
https://github.com/near/near-wallet/blob/master/src/utils/wallet.js#L12

Not relevant to this PR, but I think generally if we want to relay on URL's we should store all routes in a separate file as consts, and additionally an array of objects with properties: `path`, `component`, `exact`, so we could map through it in `Routing.js`. What do you think @vgrichina @Patrick1904 ?

File: yarn.lock [hunk 2]
```diff
@@ -9814,4 +9814,4 @@ yargs@^15.3.1:
     string-width "^4.2.0"
     which-module "^2.0.0"
     y18n "^4.0.0"
-    yargs-parser "^18.1.1"
+    yargs-parser "^18.1.1"
```
Comment (marcinbodnar) [inline yarn.lock:9817]: nit: newline at the end is missing.

File: src/middleware/index.js (hunk 1 above)
Comment (Patrick1904) [reply src/middleware/index.js]: I agree, it makes sense to organize this a bit more as number of routes/urls grows

Comment (marcinbodnar) [reply src/middleware/index.js]: OK, I added an issue, to be handled in the near future.

**Pull request merged**